java-checks-test-sources/src/main/java/symbolicexecution/checks/S2755_XxeProcessingCheck_DocumentBuilderFactory.java

```diff
@@ -165,3 +165,25 @@ DocumentBuilderFactory two_factory4(boolean b) throws ParserConfigurationExcepti
     }
   }
 }
+
+class DocumentBuilderFactoryTest_InStaticBlock {
+  private static final DocumentBuilderFactory documentBuilderFactory = DocumentBuilderFactory.newInstance();
+
+  static {
+    documentBuilderFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+    documentBuilderFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
+  }
+
+  public static DocumentBuilder noIssue() throws Exception {
+    // No issue here, the code is equivalent to the one in the other method.
```

Comment:
The "other method" is [equivalent](https://github.com/SonarSource/sonar-java/pull/3938/files#diff-61a413c2c2ca3340a7e6d425863e05ca0d559c750d012961a5520961522226f0R183)? Should we name it explicitly?
```suggestion
    // No issue here, the code is equivalent to the one in the other method {@link symbolicexecution.checks.DocumentBuilderFactoryTest_InStaticBlock#equivalent}.
```

Reply:
I like the idea, but I can not click the link (at least on my IDE). What do you think about simply:

```suggestion
    // No issue here, the code is equivalent to the one in the next method.
```

Reply:
Sound good to me 👍🏿